resume.html

```diff
@@ -0,0 +1,97 @@
+<html>
+	<head>
+	<title>
+	Resume: MICHAEL DENMAN
+	</title>
+	</head>
+
+	<body>
+		<h2>
+		<img id="Self Photo" src="self-photo.png" width="80" height="105"> MICHAEL ALEXANDER DENMAN
+		</h2>
+	325-277-2340 - denmanmd@gmail.com <a href='https://www.linkedin.com/in/michael-denman/'> LinkedIn Profile</a></br>
```

Comment:
We can add hyperlinks for both your email and phone.

Reply:
For phones, prefix the link href with `tel:` and emails with `mailto:`
Try clicking on them after those changes.

Reply:
You'll be replacing https or http, so a phone would look like `<a href="tel:+13252772340">325-277-2340</a>`